### ISSUE TITLE
Requested by & Sent by should have device name

### ISSUE THREAD

**Issue by Anushree09-N** [mosip/inji#704]
**Describe the bug**
Requested by & Sent by should have device name

**To Reproduce**
prerequisite:  VC is downloaded in the app

1. Open scanner in wallet & open qr code  in verifier
2. scan the qr code from wallet
3. select VC and share 

**Expected behavior**
-Sent by in verifier device in incoming VC screen
-Requested by in wallet in sharing screen
Above screen should have device name instead of displaying  wallet and verifier

**Actual behavior**
Requested by is always displayed as "Verifier" & sent by is always displayed as "wallet" during VC transfer.

**Screenshots**

![IMG_0224 1](https://user-images.githubusercontent.com/124761463/229486972-6b2cd671-03cd-4f39-abea-4f865f99b1c5.PNG)

![image](https://user-images.githubusercontent.com/124761463/229487102-1ea3dac8-2d9a-4c37-8f23-d08f569f61a7.png)
![Screenshot_2023-04-03-16-07-06-477_io mosip residentapp 1](https://user-images.githubusercontent.com/124761463/229493287-b5be7b3d-6e8b-4db0-8527-51a9693ca792.jpg)
![Screenshot_2023-04-03-16-09-00-756_io mosip residentapp 1](https://user-images.githubusercontent.com/124761463/229493505-1ce5c0ec-456f-4cf0-9268-2ba7d1258b0f.jpg)

**Smartphone (please complete the following information):**

- Wallet Device: 
 iphone 8 | iPhone 16 BLE 5.0

- Verifier Device: 
Redmi Note 10 Lite | Android 10 BLE 5.0

- Mimoto version: 1.2.0.1
- MOSIP Version: qa-1201-b2.mosip.net

**Where does the issue occur: Wallet/Verifier?**
Both
**Logs of wallet and verifier:**

**Comment by Raginikrishnamurthy** [comment]
device name does not exist now- descoped. closing in github.
